one_fm/public/js/doctype_js/job_opening.js

```diff
@@ -35,7 +35,7 @@ var validate_date = function(frm) {
 };
 
 var set_my_dash_board = function(frm) {
-	dashboard_link_doctype(frm, "Note", 'one_fm_job_opening');
+	//dashboard_link_doctype(frm, "Note", 'one_fm_job_opening');
```

Comment:
@pjamsheer , Could you please look into this part? It displayed some error in "Job Opening" Doc.
<img width="462" alt="Screen Shot 2022-02-20 at 1 25 56 PM" src="https://user-images.githubusercontent.com/29017559/154838276-19c9ccf0-1f28-4123-ba27-115e4383e16a.png">
.

Reply:
@SalShai  There were a requirement for creating Notes on Job Opening - Dashboard set for the same . I hope we can remove all the lines of codes related to that dashboard.